### ISSUE TITLE
Idea

### ISSUE THREAD

**Issue by Itachitard** [BabosApple/NickUI#3]
Can you guys make it so it will hide your rank too?

**Comment by BabosApple** [comment]
> Can you guys make it so it will hide your rank too?

Hi, thanks for opening issue
 We like'd to need more information about your idea/suggestion like what does rank mean it is a group from PurePerms plugin? 
 or its on the other project then PurePerms.

**Comment by Itachitard** [comment]
Yeah,pureperms,when your nick is on the group will be member or a set one and when it is off it will be the normal one that you own

**Comment by BabosApple** [comment]
> Yeah,pureperms,when your nick is on the group will be member or a set one and when it is off it will be the normal one that you own

Thank you for replying and give us a suggestion for this project!
 we will decided to start working in this update we will give you the status if we can archive your suggestion!
